docs/service-mesh/03-03-istio-rbac.md

```diff
@@ -3,22 +3,6 @@ title: Istio RBAC configuration
 type: Details
 ---
 
-As a core component, Istio is installed with Kyma by default. The ClusterRbacConfig custom resource (CR), which defines the global behavior of Istio, is created as a part of the installation process.
+>**WARNING:** Support for Istio RBAC is discontinued in version 1.5 of Istio. Use the new [Authorization Policy](https://istio.io/latest/docs/reference/config/security/authorization-policy/) instead. The new Policy was introduced in Istio 1.4. For details, read the [Introducing the Istio v1beta1 Authorization Policy blog post](https://istio.io/v1.4/blog/2019/v1beta1-authorization-policy/).
```

Comment:
```suggestion
>**CAUTION:** Support for Istio RBAC is discontinued in version 1.5 of Istio. Use the new [Authorization Policy](https://istio.io/latest/docs/reference/config/security/authorization-policy/) instead. The new Policy was introduced in Istio 1.4. For details, read the [Introducing the Istio v1beta1 Authorization Policy blog post](https://istio.io/v1.4/blog/2019/v1beta1-authorization-policy/).
```
We use **NOTE**, **TIP**, or **CAUTION**. See [here](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md#panels).